media-sound/gmusicbrowser/gmusicbrowser-1.1.15_p20160530.ebuild

```diff
@@ -0,0 +1,76 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+GIT_COMMIT="853840eb9dad0b59ad2dac5d303f5929b2f09f21"
+
+inherit eutils fdo-mime gnome2-utils
+
+DESCRIPTION="An open-source jukebox for large collections of mp3/ogg/flac files"
+HOMEPAGE="http://gmusicbrowser.org/"
+SRC_URI="https://github.com/squentin/${PN}/archive/${GIT_COMMIT}.tar.gz -> ${P}.tar.gz"
```

Comment:
Minor suggestion: I prefer if the base part of the URI is good for copy-pasting, i.e. `https://github.com/squentin/gmusicbrowser` is more convenient.